package.json

```diff
@@ -52,11 +53,13 @@
     "dom-to-image": "^2.6.0",
     "express": "^4.17.1",
     "framer-motion": "^2.9.4",
+    "geojson-validation": "^1.0.2",
     "i18next": "^19.8.3",
     "i18next-browser-languagedetector": "^6.0.1",
     "layer-manager": "^3.0.8",
     "luma.gl": "^7.3.2",
     "mapbox-gl": "^1.12.0",
+    "mapbox-gl-draw": "^0.16.0",
```

Comment:
Are the modules still used?

Reply:
Found usage of `mapbox-gl` at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/3aec7a3f96da4c32d0e47fd902c2b7db0179a50b/pages/_app.tsx#L2 but no usage of `mapbox-gl-draw`, right?

Reply:
`mapbox-gl-draw` is a required dependency of `react-mapbox-gl-draw` https://www.npmjs.com/package/react-mapbox-gl-draw
react-mapbox-gl-draw is just a react wrapper for the `mapbox-gl-draw` original mapbox library.